### PR TITLE
Add highlight when a new alternate id is added

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.shortcut.com/simpledotorg/story/6779/add-a-highlight-when-a-new-india-nhid-bp-passport-is-added

This PR will be merged in after [#3402 ](https://github.com/simpledotorg/simple-android/pull/3402)

### CODE REVIEW

**Comment by janhavisinghh** [comment]
Converted to draft as making some changes

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -33,4 +33,6 @@
 
   <color name="search_query_highlight">#52FFC800</color>
 
+  <color name="new_added_identifier_highlight">#FFF8E0</color>
```
Comment (Honey14) [inline app/src/main/res/values/colors.xml:36]: We already have this color as `simple_yellow_100` in the colors.xml, can we use that instead?

**Review by Honey14 — CHANGES_REQUESTED** [review]
LGTM, left comments, ptal

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (janhavisinghh) [reply app/src/main/res/values/colors.xml]: I've added this as a new commit @Honey14

File: app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt [hunk 2]
```diff
@@ -840,9 +840,26 @@ class EditPatientScreen : BaseScreen<
...
+      hasHighlight: Boolean,
+      alternateIdView: PatientEditAlternateIdViewBinding
+  ) {
+    if (hasHighlight) {
+      alternateIdView.alternateIdentifier.setBackgroundColor(resources.getColor(R.color.new_added_identifier_highlight))
+
+      val horizontalPadding = resources.getDimensionPixelSize(R.dimen.spacing_8)
+      val verticalPadding = resources.getDimensionPixelSize(R.dimen.spacing_4)
+
+      alternateIdView.alternateIdentifier.setPaddingRelative(horizontalPadding, verticalPadding, horizontalPadding, verticalPadding)
```
Comment (msasikanth) [inline app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt:859]: Why are we updating the padding here @janhavisinghh ? 🤔

Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt:859]: NVM, found my answer

**Review by Honey14 — APPROVED** [review]
LGTM

**Pull request merged**